client/src/App.js

```diff
@@ -17,7 +17,10 @@ function App() {
     loadFromLocal('Favourite Sights') ?? []
   );
   const [showCalendar, setShowCalendar] = useState(false);
-  const [selectDate, setSelectDate] = useState('');
+  const [selectDate, setSelectDate] = useState('28/6/2021');
```

Comment:
Is the placeholder date going to stay? :) Or is there a specific reason for a fixed date in the initial state?

Reply:
Without a placeholder date, the date for the sight that gets selected first won't get picked up for whatever reason. if you have a better idea, please let me know (: